src/main/java/core/basesyntax/strategy/OperationPerformer.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationPerformer {
+    void perform(FruitTransaction transaction, Storage storage);

```

Comment:
we can access storage directly
```suggestion
    void perform(FruitTransaction transaction);

```